Translations/ru.lproj/Localizable.strings

```diff
@@ -233,21 +233,21 @@
 // Note that the short string is limited to 30 characters and the long string is limited to 45 characters
 // Emojis cannot be used in these strings :(
 */
-"Monthly sip of coffee" = "Monthly sip of coffee";
-"Gift Keka a sip of coffee each month" = "Gift Keka a sip of coffee each month";
-"Monthly coffee" = "Monthly coffee";
-"Gift Keka a coffee each month" = "Gift Keka a coffee each month";
-"Monthly pair of coffees" = "Monthly pair of coffees";
-"Gift Keka a couple coffees each month" = "Gift Keka a couple coffees each month";
-"Monthly breakfast" = "Monthly breakfast";
-"Gift Keka a breakfast each month" = "Gift Keka a breakfast each month";
-"Tip a coffee" = "Tip a coffee";
-"Tip Keka a coffee so it can stay awake" = "Tip Keka a coffee so it can stay awake";
-"Tip a breakfast" = "Tip a breakfast";
-"Tip Keka a breakfast so it can start fresh" = "Tip Keka a breakfast so it can start fresh";
-"Tip a lunch" = "Tip a lunch";
-"Tip Keka a lunch and feed it" = "Tip Keka a lunch and feed it";
-"Tip a dinner" = "Tip a dinner";
-"Tip Keka a dinner so it keeps rolling" = "Tip Keka a dinner so it keeps rolling";
-"Tip a bunch of food" = "Tip a bunch of food";
-"Tip Keka a new bunch of food to keep rolling" = "Tip Keka a new bunch of food to keep rolling";
+"Monthly sip of coffee" = "Ежемесячный глоток кофе";
+"Gift Keka a sip of coffee each month" = "Угощать Keka глотком кофе каждый месяц";
+"Monthly coffee" = "Ежемесячный кофе";
+"Gift Keka a coffee each month" = "Угощать Keka кофе каждый месяц";
+"Monthly pair of coffees" = "Ежемесячная пара чашек кофе";
+"Gift Keka a couple coffees each month" = "Угощать Keka парой чашек кофе каждый месяц";
+"Monthly breakfast" = "Ежемесячный завтрак";
+"Gift Keka a breakfast each month" = "Угощать Keka завтраком каждый месяц";
+"Tip a coffee" = "Угостить кофе";
+"Tip Keka a coffee so it can stay awake" = "Угостить Keka кофе, чтобы не засыпать";
+"Tip a breakfast" = "Угостить завтраком";
+"Tip Keka a breakfast so it can start fresh" = "Угостить Keka завтраком для отличного начала дня";
```

Comment:
@NickKaramoff this one exceeds 45 characters so can't be used

Reply:
Fixed in #967